packages/admin/cms-admin/src/generator/future/generateGrid.ts

```diff
@@ -155,15 +106,31 @@ export function generateGrid(
 
     const showActionsColumn = allowCopyPaste || allowEditing || allowDeleting;
 
+    const {
+        imports: unsupportedRequiredGqlArgsImports,
+        props: unsupportedRequiredGqlArgsProps,
+        gqlArgs,
+    } = getXxxForUnsupportedRequiredGqlArgs([gridQueryType, ...(createMutationType ? [createMutationType] : [])]);
+    imports.push(...unsupportedRequiredGqlArgsImports);
+    props.push(...unsupportedRequiredGqlArgsProps);
```

Comment:
Why unsupported? It seems to me that you're pushing a lot of supported things?

Reply:
unsupported in a sense that these gql-args are not handled directly by the generated code and need to be addressed in the wrapping component. maybe I should call it "RequiredGqlArgsNotHandled[By|Within]GeneratedGrid"?

Reply:
- getPropsForRequiredGqlArgsNotHandledByGeneratedGrid waay too long :)

alternatives?
- getPropsForPassedThruGqlArgs
- getPropsForForwardedGqlArgs
- getForwardedGqlArgs

Reply:
[I nehm die Nummer drei!](https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DTBLy3SVPNJI&psig=AOvVaw0U6bd8gb4V6JKWt69cOrLp&ust=1714217476187000&source=images&cd=vfe&opi=89978449&ved=0CBQQjhxqFwoTCKi_tIbk34UDFQAAAAAdAAAAABAE)

I think it's good to omit the getProps because it's more than only the props. so it's the same naming-pattern as generateFormField.ts. 

getPropsForFilterProp could also be renamed to getFilterProps and generateFormField.ts could be moved into generateGrid-folder.